### PR TITLE
TRN-368: Creating Roles

### PR DESCRIPTION
**Pull request by hiyampatel**

### CODE REVIEW

File: config/views.view.content.yml [hunk 1]
```diff
@@ -176,71 +176,6 @@ display:
           settings:
             link_to_entity: true
           plugin_id: field
```
Comment (arjak-dev) [inline config/views.view.content.yml]: why this change is required?

Comment (hiyampatel) [reply config/views.view.content.yml]: No, it was not required. I removed the changes of that file.
